Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -52,17 +52,17 @@ extension JiraService {
 
         // MARK: Fields
 
-        let project = FieldType.ObjectID(id: "13402") // CRP Project
-        let issueType = FieldType.ObjectID(id: "11439") // "CRP: Code Change Request"
-        let summary: String
+        var project = FieldType.ObjectID(id: "13402") // CRP Project
```

Comment:
Why do we need this mutation? Sorry if the question doesn't make sense but I'm not finding where we need this mutability.

Reply:
It's for being able to write tests that are able to change the default CRP project – especially to make them generate the JSON that targets the [FCTP test project](https://babylonpartners.atlassian.net/secure/RapidBoard.jspa?rapidView=933&selectedIssue=FCTP-13) instead of the real CRP project.

tbh I think it's not used in the current tests (as they are written in this PR) anymore, but it was useful to generate the JSON for the CRP ticket created by using the test target but pointing to the FCTP test project, so I can then just copy/paste that generated JSON into a file and use `curl` to send it to the test project, to validate that the JIRA API understands my JSON properly and does not return any error, without the risk of accidentally post to the real CRP project during those tests.

We could revert it now that I've ran some manual tests against the FCTP project, but I figure it could be useful if we need to manually run more tests against FCTP instead of CRP in the future (also I felt it didn't harm to keep them `var` since they'd still be immutable as long as the `CRPIssue` itself was a `let` anyway)

Reply:
Thanks for the explanation @AliSoftware! I think we can keep them as they may be useful again in the future and this shouldn't be harmful as you said.